README.md

```diff
@@ -16,31 +16,54 @@ The **Platform.sh CLI** is the official command-line interface for [Platform.sh]
   * [Drush](https://github.com/drush-ops/drush) (for Drupal projects)
   * Other build tools: [npm](https://www.npmjs.com/), [pip](http://docs.python-guide.org/en/latest/starting/installation/), [bundler](http://bundler.io/), etc.
 
-## Installation
+## Install
```

Comment:
While we're here, under the top paragraph in the README should we explain what this repository represents, and link to the [platformsh/cli](https://github.com/platformsh/cli) repository?

(we'll rename this one soon, I guess to `platformsh/legacy-cli`)

Reply:
and we could do the same in the GitHub repo description, something like:

```diff
- The unified tool for managing your Platform.sh services from the command line.
+ The legacy Platform.sh CLI. For the current version see: platformsh/cli
```

Reply:
Yes, you're right @pjcdawkins . We need to make it cristal clear